server/src/main/java/io/mantisrx/server/master/store/MantisJobMetadataWritable.java

```diff
@@ -84,6 +86,9 @@ public MantisJobMetadataWritable(@JsonProperty("jobId") String jobId,
         this.name = name;
         this.user = user;
         this.submittedAt = submittedAt;
+        if(startedAt == 0) {
+            this.startedAt = submittedAt;
+        }
```

Comment:
looks like we are missing an else case here. this wont set the `this.startedAt` to the value being passed to the constructor.

Reply:
Simplified this code